### PR TITLE
Fwk 225 logging standards

### PR DESCRIPTION
**Pull request by bearfact**

### CODE REVIEW

File: initializers/logger.js [hunk 1]
```diff
@@ -1,14 +1,88 @@
...
+      };
+
+    if(!resolvedOptions.name) {
+        throw Error('Logger name not provided for logger initializer.');
+    }
+
+    const logger = bunyan.createLogger(resolvedOptions);
+    const handler = createHandler(logger);
+
+    return new Proxy(logger, handler);
```
Comment (bearfact) [inline initializers/logger.js:83]: I'm assuming this Proxy is only affecting the instanced of the logger and not the logger class itself.  If so what is the benefit of using the Proxy over creating a custom logging API in which we could seamlessly swap out the underlying logger?

File: .editorconfig [hunk 2]
```diff
@@ -1,5 +1,5 @@
 [*]
-indent_style = tab
+indent_style = space
```
Comment (bearfact) [inline .editorconfig:2]: tabs!

Comment (aprakash-gc) [reply .editorconfig:2]: oops

File: initializers/logger.js [hunk 3]
```diff
@@ -1,14 +1,113 @@
...
+	serializers: bunyan.stdSerializers,
 	stream: process.stdout,
+};
+
+const getStack = () => stackTrace.getSync().map(sf => ({
+		func: sf.functionName,
+		file: sf.fileName,
+		line: sf.lineNumber,
+		col: sf.columnNumber
+	}));
```
Comment (jan-mcsparin) [inline initializers/logger.js:17]: Nitpick - indentation is weird in this function.

**Comment by jan-mcsparin** [comment]
Looks good overall.  Need to update version of platform utils in package.json.  Most recent version (as if 5/28) is 3.14.0

**Comment by jan-mcsparin** [comment]
👍

**Comment by cat-plevak** [comment]
👍

**Pull request merged**